docs/libraries/libraries-with-cli.md

```diff
@@ -0,0 +1,669 @@
+
+To do this, [install the .NET Framework](http://getdotnet.azurewebsites.net).
+
+Additionally, if you wish to support older targets, you will need to install targeting/developer packs for older framework versions from the [target platforms page](http://getdotnet.azurewebsites.net/target-dotnet-platforms.html).  Refer to this table:
+
+| .NET Framework Version | Thing to download |
+| ---------------------- | ----------------- |
+| 4.6 | .NET Framework 4.6 Targeting Pack |
+| 4.5.2 | .NET Framework 4.5.2 Developer Pack |
+| 4.5.1 | .NET Framework 4.5.1 Developer Pack |
+| 4.5 | Windows Software Development Kit for Windows 8 |
+| 4.0 | Windows SDK for Windows 7 and .NET Framework 4 |
+| 2.0 and 3.5 | .NET Framework 3.5 SP1 Runtime (or Windows 8+ version) |
+
+## How to target .NET Core
+
+For the purposes of building a library, targeting ".NET Core" means targeting the .NET Platform Standard.  To see what that means exactly, see [.NET Platform Standard](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md).
+
+Note that as of this time, `netstandardX.X` is not supported.  You'll have to use the `dotnet` monikers.  Here's a mapping:
+
+| dotnet | netstandard |
+|--------| ----------- |
+| dotnet51 | netstandard1.0 |
+| dotnet52 | netstandard1.1 |
+| dotnet53 | netstandard1.2 |
+| dotnet54 | netstandard1.3 |
+| dotnet55 | netstandard1.4 |
+
+
+The following is what you need to know about .NET Platform Standard targeting:
+
+You need to pick a version of `dotnetXX` (where `XX` is a version number) to add to your `project.json` file.
+
+1. You acn target the latest version - `dotnet55` - of the .NET Platform Standard, which will give you access to the most APIs.  `dotnet55` is supported by .NET Framework 4.6 (and higher) and .NET Core 1. 
+2. You can target any earlier version of the .NET Platform Standard to target earlier .NET platforms and versions.  Earlier .NET Platform Standard versions expose fewer APIs. Example targets:
+
+    - .NET Framework 4.6, 4.5.2, 4.5.1, or 4.5
+    - Windows Phone
+    - Windows Phone Silverlight
+    - Universal Windows Platform
+    - DNX Core
+    - Xamarin Platforms
+    - Mono 
+
+    [Refer to the platform mapping table](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md#mapping-the-net-platform-standard-to-platforms) to choose the `dotnetXX` moniker you need to target.
+    
+    For example, if you wanted to have compatibility with .NET Core and .NET Framework 4.6, you would pick `dotnet54`.
+
+    ```javascript
+    {
+        "frameworks":{
+            "dotnet54":{}
+        }
+    }
+    ```
+    
+    If you wanted to be compatible with Windows Phone Silverlight 8, you would pick `dotnet51`.  Note that this gives you the ability to target more platforms that just Windows Phone Silverlight 8, but at the cost of less available APIs.
```

Comment:
Try to use the word "target" instead of "compatible". Compatibility has more meaning than we want to use here.